system_metrics_collector/src/system_metrics_collector/linux_process_memory_measurement_node.cpp

```diff
@@ -60,6 +62,25 @@ LinuxProcessMemoryMeasurementNode::LinuxProcessMemoryMeasurementNode(
 {
 }
 
+LinuxProcessMemoryMeasurementNode::LinuxProcessMemoryMeasurementNode(
+  const rclcpp::NodeOptions & options)
+: PeriodicMeasurementNode(
+    "linuxProcessMemoryCollector",
+    system_metrics_collector::kDefaultCollectPeriod,
+    system_metrics_collector::kStatisticsTopicName,
+    system_metrics_collector::kDefaultPublishPeriod),
+  pid_(std::to_string(GetPid())),
+  file_to_read_(kProc + std::to_string(GetPid()) + kStatm)
+{
+  const auto r = rcutils_logging_set_logger_level(get_name(), RCUTILS_LOG_SEVERITY_DEBUG);
```

Comment:
Setting the log to debug isn't needed. Let's allow a user to set that configuration and remove our testing code.

Reply:
Done.